src/main/kotlin/uk/gov/dluhc/printapi/database/mapper/PrintRequestStatusDtoMapper.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.dluhc.printapi.database.mapper
+
+import uk.gov.dluhc.printapi.database.entity.PrintRequestStatus
+import uk.gov.dluhc.printapi.dto.PrintRequestStatusDto
+
+class PrintRequestStatusDtoMapper {
```

Comment:
I've renamed the mapper class - it looks like a delete and new class, but its just a rename